galette/templates/default/pages/configuration_dynamic_translations.html.twig

```diff
@@ -46,12 +46,17 @@
                     </table>
                 </div>
             </div>
+    {% if mode != 'ajax' %}
             <div class="ui basic center aligned segment">
                 <button type="submit" name="trans" class="ui labeled icon primary button action">
                     <i class="save icon"></i> {{ _T("Save") }}
                 </button>
-                {% include "components/forms/csrf.html.twig" %}
             </div>
+    {% else %}
+            <input type="hidden" name="trans" id="trans" value="true"/>
+            <input type="hidden" name="redirect_uri" id="redirect_uri" value="PATH"/>
```

Comment:
`PATH` seems wrong here

Reply:
This condition is for ajax requests. The value should effectively be empty as it it set in js before submit.